### ISSUE TITLE
Add a way to filter by page in the translation manager

### ISSUE THREAD

**Issue by jgramp** [andrewhancox/moodle-filter_translations#36]
It would be useful to be able to see all "System" translations, or all "Front page" translations for example, so you can work through updating the front page first for example.

Is there a way to enable filtering in the table? This would also help filter other columns (e.g. language, Issue/Status, Original content maybe as a search?)

![image](https://user-images.githubusercontent.com/8575694/159593381-67b424a7-7950-4c37-a62f-9166ebb7af60.png)

**Comment by andrewhancox** [comment]
No... but we could consider logging each time a translation gets used in the same way we track stale/missing translations.

**Comment by annakrassa** [comment]
I would love to have a search field that would search key words from the Original content column...

**Comment by andrewhancox** [comment]
You can either search by page url or navigate to the context and then use the new stale/missing in this course menu items.
